cppfd/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ include_directories(${CMAKE_CURRENT_BINARY_DIR})
 
 # Setting options
 option(OUTPUT_VTK_FILES "When OFF allows for a non-VTK build" OFF)
+option(USE_MPI "When OFF allows for a non-MPI build" OFF)
```

Comment:
With this being `OFF` by default, this feature will not get tested in CI.
Long-term it would be really good to have that configuration tested as well.